src/Take.Blip.Builder/LazyInput.cs

```diff
@@ -66,14 +66,22 @@ public class LazyInput
                     return envelopeSerializer.Serialize(Message);
                 }
 
-                return null;                
+                return null;
             });
         }
 
         public Message Message { get; }
-        
+
         public Document Content => Message.Content;
-        
+
+        public bool Analyzable {
+            get {
+                string result = null;
+                Message?.Metadata?.TryGetValue("ai.analyzable", out result);
```

Comment:
I known that I suggested the name, but I want to suggest a little change :)

This metadata is not used by the "ai" extension but by the builder. So I think `builder.analyzable` is a better fit.